bindings/rust/Cargo.toml

```diff
@@ -3,18 +3,13 @@ name = "c-kzg"
 version = "0.1.0"
 edition = "2021"
 license = "Apache-2.0"
-# XXX: Note that we don't set a `links` attribute for Cargo, because this library may link
```

Comment:
since we no longer have to do this hack, I think we can restore `links = "ckzg"` here